.gitignore

```diff
@@ -1,12 +1,12 @@
-.DS_Store
+__pycache__/
 .cache
-.tox
 .coverage
-records.json
-*.pyc
-__pycache__/
+.DS_Store
 *.egg-info/
 *.egg
+*.pyc
+*.sw[op]
+Pipfile.lock
```

Comment:
I'm not sure what's happened, but this adds lines that are already in master. See https://github.com/Kinto/kinto-integration-tests/blob/master/.gitignore#L9 for example.